core/src/main/java/org/apache/accumulo/core/util/CleanUp.java

```diff
@@ -18,22 +18,34 @@
 
 import java.util.Set;
 
-import org.apache.accumulo.core.client.impl.ThriftTransportPool;
-import org.apache.accumulo.fate.zookeeper.ZooSession;
+import org.apache.accumulo.core.client.AccumuloClient;
+import org.apache.accumulo.core.singletons.SingletonManager;
+import org.apache.accumulo.core.singletons.SingletonManager.Mode;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+/**
+ * This class was created because Connector was used static resource that created threads. There was
+ * no way to clean these resource up other than using this class. The new {@link AccumuloClient} API
+ * is closable. For code than only uses AccumuloClient, when all AccumuloClients are closed then
```

Comment:
Typo? "than"... I think this sentence could be more clear.

Reply:
I reworded the docs.